HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py

```diff
@@ -265,7 +265,7 @@
             "HLT_L1_TripleJet_VBF_v"
             ),
         recJetLabel  = cms.string("ak4PFJetsCHS"),
-        jetTagLabel  = cms.string("pfCombinedSecondaryVertexBJetTags"),
```

Comment:
@ferncek The intention is to monitor paths and thus also the btagging at HLT level. If pfCombinedInclusiveSecondaryVertexV2BJetTags is the default one used offline it makes sense to use this one for the dqm. 
Definitely if IVF is also used for HLT b-tagging, so it would be closer to the algorithm used at HLT level. @silviodonato Is this the case?